observation/batch_reader.go

```diff
@@ -0,0 +1,36 @@
+package observation
+
+// BatchReader provides an interface to read Observations
+type BatchReader interface {
```

Comment:
Is the batch reader actually used? I can see it mentioned in comments in the CSVReader, but NewBatchReader is never called